### ISSUE TITLE
Price Alert window and tooltips

### ISSUE THREAD

**Issue by TechsUsInc** [bitshares/bitshares-ui#2204]
**Describe the bug**
1. Initially upon first time clicking the Price Alert, the pop up window appears void of data in the price 
    field.
![init_price_alert_capture](https://user-images.githubusercontent.com/18186497/48953269-e249d100-ef0a-11e8-9dd3-e3bc58e28178.JPG)

   Closing and reopening the window shows a price in the price field. 
![init_price_alert_2_capture](https://user-images.githubusercontent.com/18186497/48953284-eb3aa280-ef0a-11e8-9296-a3ce86adf4b5.JPG)

2. Opening the Price Alert window blocks the asset chart and the prices cannot be seen without closing 
    the window. The user either has to copy/paste their target price or remember/write it down prior to 
    setting the alert. Consider moving slightly to the left to reveal the chart pricing while the Price Alert 
    window is open
3. Price Alert window tooltips unresponsive

**To Reproduce**
1. Go to account, click on Exchange tab
2. Click on an asset pair (BTS:Open.BTC) 
3. Click on the Price Alert icon
4. Note empty price field
5. Click cancel closing the Price Alert window
6.  Reopen the Price Alert window by clicking the Price Alert icon
7.  Note data now present in the price field
8.  Note that the Price Alert window blocks the chart prices
9.  With the Price Alert window open, hover over the first asset in the pair. 
10. Note that a question mark tooltip appears. 
11. Hover over the second asset in the pair 
12. Note that there is no tooltip or the question mark

**Expected behavior**
1. Ability to see the chart price with the Price Alert window open
2. Automatically suggested price in the price field of the Price Alert window
3. Tooltip explaining what the Price Alert icon is for
4. Tooltips within the Price Alert window to function
   
**Desktop (please complete the following information):**
 - OS: Windows 8.1
 - Browser Opera

**Additional context**
Note:
1. With Devtools open, the Price Alert window is left aligned enough to see the prices within the chart
![init_price_alert_3_capture](https://user-images.githubusercontent.com/18186497/48954695-428f4180-ef10-11e8-8896-22d90f046a04.JPG)

2. Engaging with Devtools enabled the suggested price in the Price Alert field.
3. From within the Price Alert window, clicking on or engaging devtools enabled the asset pair tooltip 
   but only when the first asset is clicked or hovered over.

**Comment by gibbsfromncis** [comment]
@TechsUsInc if you click "add rule" it puts in the field the "latest" price. Doesn't it?

**Comment by TechsUsInc** [comment]
When I clicked on Price Alert icon, the window opened with a empty price field. Clicking inside it requires a manual entry.
When I clicked "add rule" it does indeed add the price into the price field, however, there are now two price fields. One is empty, the other pre-filled. 
![init_price_alert_4_capture](https://user-images.githubusercontent.com/18186497/48962485-2f976400-ef46-11e8-98fb-f8c304653e90.JPG)

**Comment by gibbsfromncis** [comment]
@TechsUsInc Yep, I agree with you about that and this is confusing thing that I'm going to fix. 
What do you think, Is it nice to fill Price input by latest price value when user click add rule?

**Comment by TechsUsInc** [comment]
The price alert is an awesome feature. 
I think the auto filled price field is very nice as long as the user can still see the prices in the chart allowing them to alter the price field at will, and see the chart, while in the Price Alert window without exiting.

**Comment by TechsUsInc** [comment]
Thinking about it further. I noticed that regardless of the choice selected (lower than/higher than) the price automatically placed in the price field are the same. Perhaps one should point to the buy price, and the other point to the sell price?
![init_price_alert_5_capture](https://user-images.githubusercontent.com/18186497/48962752-6ec7b400-ef4a-11e8-9056-f094f540f937.JPG)
In this image, the latest 24 hour price can be seen above the Price Alert window (wording latest 24 hr covered up by the Price Alert window while in Devtools, otherwise OK normally). It can also  be seen in yellow on the chart to the right of the Price Alert window (Yellow price can be seen while in Devtools, otherwise covered up normally).
I input two different prices in the fields in attempt to reflect auto pointing to a buy price and a sell price.
Would it be better to have the price field auto fill according to the request? As it is now, either way a manual entry may be required. I don't find it useful to know the current lower than/higher than because I prefer a larger spread in alerts informing me of my set high target, and set low target. As it is now, an asset should move up or down and that alert could be triggered or non applicable before the window is even saved or immediately right after because they are too close together.

**Comment by sschiessl-bcp** [comment]
I see your intended usage. The latest price is always filled even when opening first in actual release.

My suggestion:
 - On opening and on changing the left dropdown: If user has not altered the price yet, "Lower Than"/"Higher Than" fills in the respective sides from the spread
 - Display the correct price in the headline, or display it in each line for clarity (it only says "BTS" instead of "BTS/open.BTC". In the example above you show the market open.BTC/BTS and below the BTS/open.BTC price.
 - Condense the whole view (label as small as possible, fit to content). Add a new element somewhere to manipulate the price in %, here is my UX (apologies for my poor color/layout choice, but you get the idea):
![image](https://user-images.githubusercontent.com/33128181/48998126-c89aca80-f152-11e8-8b18-42a9f2b1212d.png)
The plus minus and number could also be in a new line and completely horizintally aligned if space becomes an issue

**Comment by gibbsfromncis** [comment]
@sschiessl-bcp @startailcoon do you want to take this one?

**Comment by sschiessl-bcp** [comment]
Would this Modal be transitionable to new layout?

**Comment by gibbsfromncis** [comment]
@sschiessl-bcp I'm not sure... Functionality - yes. Interface - Maybe...

**Comment by sschiessl-bcp** [comment]
Thanks, will assess
